File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.cpp
#include <iostream>
#include <string>

using namespace std;
int solution(int n)
{
    
    int sum = 0;
    string numberStr = to_string(n);
    
    for(int i =0; i< numberStr.length(); i++){
        sum +=  numberStr[i] - '0';
    }
    
    int answer = sum;

    cout << answer;
    return answer;
}

File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.java
import java.util.*;

public class Solution {
    public int solution(int n) {
        int answer = 0;
       
        String str = String.valueOf(n);
    
        System.out.println(str);    
        
        for(char ch: str.toCharArray()){
            answer += Integer.parseInt(String.valueOf(ch));
        }
        
        return answer;
    }
}